### DISCUSSION TITLE
Unable to use user dedicated IP ranges in projects

### DISCUSSION THREAD

**Discussion by cdfgallo** [apache/cloudstack#10110] [Q&A] [answered]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI, Networking
~~~

##### CLOUDSTACK VERSION

~~~
4.19.1.3
~~~

##### CONFIGURATION

Advanced networking

##### OS / ENVIRONMENT

N/A

##### SUMMARY

If you insert and IP range dedicated to a user in a Public traffic type, this is not visible in the project view, and not usable.
I don't know if this is a bug or the correct behavior. I know that I could assign an IP range inside a single project to achieve this, but I find that this leads to a worsening of the usage since the domadmin has no possibility to manage the physical networks of a zone.

##### STEPS TO REPRODUCE

~~~
Create and assign an IP range dedicated to a domadmin of a non-root domain in the default view.
From the domadmin account create a project.
Create an isolated network and see the public IP address assigned to the VR
~~~

##### EXPECTED RESULTS

~~~
The public IP addresses assigned to the accounts should be usable also in projects.
~~~

##### ACTUAL RESULTS

~~~
The public IP range is not present and not usable inside the project
~~~

**Comment by weizhouapache** [comment, marked as answer]
do not dedicate to the domain admin 
- in default view, dedicate to a domain (ips can be used by all projects/accounts in the domain)
- in project view, dedicate to a domain (ips will be dedicated to the project)
